### PR TITLE
wasm-decompile: Avoid trailing whitespace

### PR DESCRIPTION
**Pull request by relrelb**

### CODE REVIEW

File: src/decompiler.cc [hunk 1]
```diff
@@ -149,12 +149,13 @@ struct Decompiler {
     auto& right = args[1];
     BracketIfNeeded(left, precedence);
     BracketIfNeeded(right, precedence);
-    auto width = infix.size() + left.width() + right.width();
+    auto width = infix.size() + left.width() + right.width() + 2;
     if (width < target_exp_width && left.v.size() == 1 && right.v.size() == 1) {
-      return Value{{left.v[0] + infix + right.v[0]}, precedence};
+      return Value{{left.v[0] + " " + infix + " " + right.v[0]}, precedence};
```
Comment (aardappel) [inline src/decompiler.cc:154]: can use `cat` here too

File: src/decompiler.cc [hunk 2]
```diff
@@ -474,7 +475,7 @@ struct Decompiler {
         auto& se = *cast<StoreExpr>(n.e);
         LoadStore(args[0], n.children[0], se.offset, se.opcode, se.align,
                   se.opcode.GetParamType2());
-        return WrapBinary(args, " = ", true, Precedence::Assign);
+        return WrapBinary(args, "=", true, Precedence::Assign);
```
Comment (aardappel) [inline src/decompiler.cc:478]: So there were no calls to `WrapBinary` that don't want the spacing?

**Review by aardappel — COMMENTED** [review]
Thanks, looks like a nice improvement!

File: src/decompiler.cc (hunk 2 above)
Comment (relrelb) [reply src/decompiler.cc:478]: Yup, all 3 calls had spaces before and after.

File: src/decompiler.cc (hunk 1 above)
Comment (relrelb) [reply src/decompiler.cc]: Done.

**Review by aardappel — APPROVED** [review]

**Comment by aardappel** [comment]
Thanks!

**Pull request merged**